rules/windows/process_creation/proc_creation_win_powershell_console_history_file_access.yml

```diff
@@ -0,0 +1,23 @@
+title: Possible PowerShell Console History File Access Attempt
+id: f4ff7323-b5fc-4323-8b52-6b9408e15788
+status: experimental
+description: Detects possible access attempts to the PowerShell console history file (ConsoleHost_history.txt) in order to retrieve plaintext passwords used in PowerShell commands.
```

Comment:
```suggestion
description: Detects possible access attempts to the PowerShell console history directly via history file (ConsoleHost_history.txt). This can give access to plaintext passwords used in PowerShell commands or used for general reconnaissance.
```